course_content/iris_course/3.Subcube_Extraction.ipynb

```diff
@@ -331,7 +331,7 @@
    "source": [
     "Combining multiple constraints can be done in couple of ways. One way involves creating _a single Constraint instance_ with multiple requirements. \n",
     "\n",
-    "For example, using our original CubeList, we extract a cube named `air_potential_temperature` with model level number 10."
```

Comment:
In this case, AFAICS it _does_ need to be 'air_potential_temperature'.
This data, as originally loaded, is just (awkwardly) different.

Reply:
Thanks, have changed back. See [this commit](https://github.com/SciTools/courses/pull/177/commits/bd34c93bfb9945f4d1d90770f6534dcb414645ce)